FAQ.md

```diff
@@ -2,95 +2,137 @@
 [For better viewing, including hyperlinks, read it online at ]:#
 [https://github.com/kvz/bash3boilerplate/blob/master/FAQ.md  ]:#
 
-## Contents
+# Frequently Asked Questions
```

Comment:
I think the reason this was done was for the menu's to work on the live site, it's a bit of a quick-hack job that relies on hiding this content section, adding it to the menu:

![screen shot 2016-11-22 at 09 58 44](https://cloud.githubusercontent.com/assets/26752/20517196/4c635c02-b09a-11e6-9700-220eb0569a8c.png)

The rest of this change seems to be a mix of formatting changes that seem to be personal taste but could break linting, as well as content changes that are make this document more correct. I think the later could be merged in easily, but this PR in a whole is shaky for the current (admittedly fragile/horrible) build process.